### PR TITLE
Refactor useNativeAssetAsWrappedAmountIn into wethIsEth

### LINKED ISSUE #227: Refactor useNativeAssetAsWrappedAmountIn into wethIsEth
There are a few subtasks on this: 

1. rename `useNativeAssetAsWrappedAmountIn` to `wethIsEth`
2. rename `toNativeAsset` to `wethIsEth`
3. move `wethIsEth` from AddLiquidity `query` to `buildCall` interface
4. move `wethIsEth` from RemoveLiquidity `query` to `buildCall` interface

Note: v3 Router interface for Add/Remove Liquidity does not expose a `wethIsEth` input for query operations, but only for the actual transactions, so items 3 and 4 aim to make v2 interface follow the same standard. The assumption is that using ETH or WETH should not affect the query result.

### PR DESCRIPTION
**Pull request by brunoguerios**
Closes #227

### CODE REVIEW

File: test/v2/removeLiquidity/composableStable.recovery.integration.test.ts [hunk 1]
```diff
@@ -98,6 +98,7 @@ describe('composable stable remove liquidity test', () => {
                 txInput.slippage,
             );
         });
+        // TODO: with native asset
```
Comment (johngrantuk) [inline test/v2/removeLiquidity/composableStable.recovery.integration.test.ts:101]: Worth adding now?

File: test/v2/removeLiquidity/weighted.recovery.integration.test.ts [hunk 2]
```diff
@@ -103,6 +103,7 @@ describe('weighted remove liquidity recovery test', () => {
                 txInput.slippage,
             );
         });
+        // TODO: with native asset
```
Comment (johngrantuk) [inline test/v2/removeLiquidity/weighted.recovery.integration.test.ts:106]: Now?

**Review by johngrantuk — COMMENTED** [review]
Definitely seems like better/easier to follow terminology. Swaps are still using `wethIsEth` should that also be taken care of in this PR to maintain consistency?

File: test/v2/removeLiquidity/composableStable.recovery.integration.test.ts (hunk 1 above)
Comment (brunoguerios) [reply test/v2/removeLiquidity/composableStable.recovery.integration.test.ts:101]: I'd say yes - will add it here 👍

**Comment by brunoguerios** [comment]
Nice! 
Yeap, I split initially to reduce complexity of each commit and forgot about swaps 😅
It makes total sense to update that in this PR as well - will do that now 👍

**Comment by brunoguerios** [comment]
After discussing with the team, Daniel suggested we kept `wethIsEth` for consistency with smart contracts and also across all interfaces (add/remove/swap).
Current changes already take that into account.

File: src/entities/addLiquidity/types.ts
Comment (johngrantuk) [inline src/entities/addLiquidity/types.ts:1]: Not sure if its already covered elsewhere but should we take this opportunity to refactor `Add/RemoveLiquidityQueryOutput` to also return chainId so it doesn't have to be passed to build?

**Review by johngrantuk — APPROVED** [review]

File: src/entities/addLiquidity/types.ts
Comment (brunoguerios) [reply src/entities/addLiquidity/types.ts:1]: done ✅

**Pull request merged**